components/changelog/ChangelogTrigger.js

```diff
@@ -0,0 +1,93 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { graphql } from '@apollo/client/react/hoc';
+import themeGet from '@styled-system/theme-get';
+import styled from 'styled-components';
+
+import { API_V2_CONTEXT, gqlV2 } from '../../lib/graphql/helpers';
+import { parseToBoolean } from '../../lib/utils';
+
+import Container from '../Container';
+import { Flex } from '../Grid';
+import { withNewsAndUpdates } from '../NewsAndUpdatesProvider';
+import { Dropdown } from '../StyledDropdown';
+import { useUser } from '../UserProvider';
+
+import ChangelogNotificationDropdown from './ChangelogNotificationDropdown';
+
+const CHANGE_LOG_UPDATES_ENABLED = parseToBoolean(process.env.CHANGE_LOG_UPDATES_ENABLED);
+
+const FlameIcon = styled(Flex)`
+  border-radius: 50%;
+  background-color: ${props => themeGet(props.backgroundColor)};
+  background-image: ${props => `url(${props.url})`};
+  background-repeat: no-repeat;
+  background-position: center center;
+  height: 30px;
+  width: 30px;
+  margin-left: 2px;
+`;
+
+const ChangelogTrigger = props => {
+  const { setShowNewsAndUpdates, setChangelogViewDate } = props;
+  const [showChangelogDropdown, setShowChangelogDropdown] = useState(true);
+  const { LoggedInUser, refetchLoggedInUser } = useUser();
+  const hasSeenNewUpdates = LoggedInUser?.hasSeenLatestChangelogEntry;
+
+  const handleShowNewUpdates = async () => {
+    setShowNewsAndUpdates(true);
+    await setChangelogViewDate({ variables: { changelogViewDate: new Date() } });
+    refetchLoggedInUser();
+  };
+
+  useEffect(() => {
+    if (LoggedInUser) {
+      refetchLoggedInUser();
+    }
+  }, []);
+
+  return (
+    <React.Fragment>
+      {LoggedInUser && CHANGE_LOG_UPDATES_ENABLED && (
```

Comment:
:ice_cream: It's maybe simple to have that outside of the `return`, it prevents having to use a Fragment:

```es6
if (!LoggedInUser || !CHANGE_LOG_UPDATES_ENABLED) {
  return null;
}
```

Reply:
Done. Thanks. 😄